docs/materials/htc/part1-ex3-jobs.md

```diff
@@ -201,7 +201,7 @@ or perhaps a simple shell script of commands that you'd like to run within a job
         In other words, in this format the executable must be in the same directory as the submit file.
 
     !!! note
-        As this example shows, blank lines and spaces around the = sign do not matter to HTCondor.
```

Comment:
What’s wrong with calling out that the example demonstrates the point? It does not make the statement less general. I’m fine with the change to the = sign thing, but I do not understand the value of the first part of the change.

Reply:
I'm being (perhaps overly) pedantic; the example doesn't show the "spaces around `=`" part. I don't know that the callback to the example adds much, and this makes the text easier to move around or edit in the future.

Reply:
Then make the example show what we mean! (Is my opinion…)

Reply:
A thought: putting spaces around `=` is for-sure good practice. Maybe we just don't even mention it, and bring back the "as this example shows"? Speak no evil, hear no evil, see no evil...

Reply:
Added an explicit example in the note showing the behavior.